inc/class-cachify.php

```diff
@@ -134,21 +134,14 @@ public function __construct() {
 
 		/* Backend */
 		if ( is_admin() ) {
-			add_action(
-				'wpmu_new_blog',
-				array(
-					__CLASS__,
-					'install_later',
-				)
-			);
-
-			add_action(
-				'delete_blog',
-				array(
-					__CLASS__,
-					'uninstall_later',
-				)
-			);
+			if ( version_compare( get_bloginfo( 'version' ), '5.1', '<' ) ) {
+				// The following hooks are deprecated since WP 5.1 (#246).
+				add_action( 'wpmu_new_blog', array( __CLASS__, 'install_later' ) );
+				add_action( 'delete_blog', array( __CLASS__, 'uninstall_later' ) );
+			} else {
+				add_action( 'wp_initialize_site', array( __CLASS__, 'install_later' ) );
```

Comment:
Do we agree that [`wp_initialize_site`](https://developer.wordpress.org/reference/hooks/wp_initialize_site/) is the correct hook here?

We used to use [`wpmu_new_blog`](https://developer.wordpress.org/reference/hooks/wpmu_new_blog/) which "fires immediately after a new site is created".
The recommended replacement is [`wp_insert_site`](https://developer.wordpress.org/reference/hooks/wp_insert_site/) which "fires once a site has been inserted into the database".

We do add empty options (i.e. all default) into the database. Can be seen as part of the "initialization routine", so the initialize-hook might be the semantically better choice here. Just wanna make sure this is done intentionally.